plugin/src/main/java/io/jenkins/plugins/casc/impl/DefaultConfiguratorRegistry.java

```diff
@@ -84,12 +83,7 @@ public Configurator lookup(Type type) {
 
     private LoadingCache<Type, Configurator> cache = Caffeine.newBuilder()
             .expireAfterAccess(10L, TimeUnit.SECONDS)
-            .build(new CacheLoader<Type, Configurator>() {
-                @Override
-                public Configurator load(@NonNull Type type) throws Exception {
-                    return internalLookup(type);
-                }
-            });
+            .build(type -> internalLookup(type));
```

Comment:
This is getting rid of the cache part of this lookup. Which seems bad.

Reply:
After a closer look, the Caffeine part is still there with the cache, I was wrong 😅

This can definitely be cleaned up.